### PR TITLE
Feature: loading screen

### PR DESCRIPTION
**Pull request by AwesomestCode**
* Add image
* Change webpack to make testing new HTML w/o webpacking easier

### CODE REVIEW

**Comment by rom1504** [comment]
Ready for review ?

**Comment by AwesomestCode** [comment]
There are still some minor issues I need to fix.

**Review by rom1504 — CHANGES_REQUESTED** [review]
can you remove the js from index.html ? it creates bugs

**Comment by AwesomestCode** [comment]
I gave up on fixing the kicked event handler. @Karang said it works fine for him, which is very confusing, since it doesn't work for me. If you can fix it, please do, but I'm giving up on this for today to start working on inventory.

File: index.js [hunk 1]
```diff
@@ -21,18 +38,37 @@ async function main () {
...
   })
 
+  bot.once('error', () => {
+    console.log('Encountered error!')
+    status = 'Error encountered. Please reload the page'
+  })
+
+  bot.once('kicked', () => {
+    console.log('User was kicked!')
+    status = 'The Minecraft server kicked you. Please reload the page to rejoin'
```
Comment (u9g) [inline index.js:59]: this isnt sent because it encounters an error prior to this, just put the message user was disconnected on the bot.once('end') event

Comment (AwesomestCode) [reply index.js:59]: Even after changing this, it still doesn't work. I'm giving up on this for now. It's going to be pretty rare of an issue, and spending any more time which could be used on adding crucial features such as inventory isn't really worth it any more.

**Comment by rom1504** [comment]
Can you rebase on master ?

**Review by rom1504 — APPROVED** [review]

**Comment by AwesomestCode** [comment]
Rebased, didn't get a chance to get to test. Maybe I messed up, please check for me.

**Comment by rom1504** [comment]
Lint failing

**Comment by AwesomestCode** [comment]
Fixed, ready for review.

**Pull request merged**